Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM quay.io/azavea/postgis:latest
```

Comment:
Addressed in https://github.com/azavea/pfb/pull/15/commits/63aeaf1f26162a38d0f18987670e4f6e88f32a59